NEWS.md

```diff
@@ -4,6 +4,8 @@
 
 * The package will now log a backtrace for errors and warnings that occur during tuning. When a tuning process encounters issues, see the new `trace` column in the `collect_notes(.Last.tune.result)` output to find precisely where the error occurred (#873).
 
+* When automatic grids are used, [dials::grid_space_filling()] is now used (instead of [dials::grid_latin_hypercube()]).
```

Comment:
Do we know if this markup work in news files?

Reply:
Downlit will automatically link to docs when using backticks for pkgdown changelogs—let's use backticks in favor of the Roxygen2 formatting.